plex_trakt_sync/config.py

```diff
@@ -27,6 +47,8 @@ def __getitem__(self, item):
         return dict.__getitem__(self, item)
 
     def initialize(self):
+        self.update(self.defaults)
+
         if not exists(config_file):
             with open(default_config_file, "r") as fp:
                 defaults = json.load(fp)
```

Comment:
Doesn't your addition make the default config file obsolete? If so, doesn't the `if not exists...` load the default config file and save it as the new config on the first run without a config file, overriding `self.defaults` in every future execution?

Lines 55 and 56 are also part of the behavior I am describing but I could not select them.

Reply:
@Taxel this code is executed only if the config file does not exist.

But you gave me an idea of how to refactor this: must load the default config ourselves, and then user config.

Reply:
Fixed in f94bc56